vizro-core/tests/unit/vizro/actions/_action_loop/test_get_action_loop_components.py

```diff
@@ -141,7 +141,7 @@ def test_no_components(self):
         ],
         indirect=True,
     )
-    def test_all_action_loop_components(  # pylint: disable=too-many-arguments  # noqa: PLR0913
+    def test_all_action_loop_components(  # pylint: disable=too-many-arguments
```

Comment:
```suggestion
    def test_all_action_loop_components(
```

Reply:
But you are right that this ignore is not needed. Let me actually remove all of the #pylint ignores that we might not need anymore 👍